cis/cis_requests.py

```diff
@@ -69,14 +69,26 @@ def get_cui_categories(file, config):
     return []
 
 def get_eml_file(email_id, file_name, access_token, config):
-  headers = {"Content-Type": "application/json", "Authorization": "Bearer " + access_token}
-  body = {"filename":file_name, "emailid":email_id}
-  r = requests.get("http://" + config.ezemail_server + "/ezemail/v1/getemlfile", data=json.dumps(body), headers=headers, timeout=30)
-  if r.status_code == 200:
-    return r.content
+
+  def get_eml_file_archive(email_id, file_name, access_token, config):
```

Comment:
Could you define this method outside of the get_eml_file method? Usually you don't want to define a method inside another if you can avoid it.